client/src/components/ButtonsBar/style.scss

```diff
@@ -1,4 +1,35 @@
-.buttons-container {
-  display: inline-block;
-  z-index: 1;
+@import "@/constants/style";
+
+$bar-width: 14rem;
+$bar-height: 3rem;
+
+.buttons-bar-wrapper {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  align-items: end;
+  justify-content: center;
+
+  .buttons-bar {
+    background: rgba(0, 0, 0, 0.73);
+    padding: 0.3rem 1rem;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    align-items: center;
+    border-radius: 5rem;
```

Comment:
Would be nice to have default border-radius or something

Reply:
That's true I'll move it into the constants